HumanExposure/factotum
# 1675: Bulk Assign PUCs for Predicted Products

1. On /bulk_product_puc page, filter Products by no PUC assigned or only AU PUC assigned.
2. Added Predicted PUC Assignment column to display the AU PUC is one exists
To test locally:
1. Go to /bulk_product_puc page, search by "Product 5", Product 50 should have Predicted PUC Assignment value.
2. Select Product 5 and assign a PUC to it, then Product 5 should disappear from the Product table.
